### PR TITLE
studio: further enhancements

### PR DESCRIPTION
**Pull request by tapadipti**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/studio/create-view.md
Comment (duijf) [inline content/docs/studio/create-view.md:90]: If there is time: it might also be good to document the steps that users can take on GitHub + add screenshots.

It might also be worthwhile to note that users can only give us access to the repositories they want to use with Studio. They don't have to grant us access to all repositories in their GitHub org if they don't want to.

**Review by duijf — COMMENTED** [review]
Thanks for documenting the GitHub app stuff @tapadipti! Looks good to me!

I have added two suggestions for things that we could mention in addition.

File: content/docs/studio/create-view.md
Comment (tapadipti) [reply content/docs/studio/create-view.md:90]: Thanks @duijf I've added a note regarding limiting the access to only the required repositories. 
Regarding documenting the steps to be taken on GH, I was thinking that since that part is external to us, maybe we should not add details/screenshots for that, coz we don't have control on if/when there's any change in that. What do you think?

Comment (tapadipti) [reply content/docs/studio/create-view.md:90]: Added notes below for limiting access.

Comment (duijf) [reply content/docs/studio/create-view.md:89]: Cool! Thanks :)

Regarding screenshots: I feel like users might benefit from the additional docs more than what this would cost us in terms of maintenance.

While the UI on GitHub's side may change slightly, the main concepts should continue to apply. (If the flow changes in major ways, they will probably announce this somehow). Having the screenshots be slightly outdated won't be the end of the world I think + it's relatively quick to fix after we find out.

Your call though!

Comment (tapadipti) [reply content/docs/studio/create-view.md:89]: Got it. I'll create a follow-up ticket to add more GH instructions/screenshots, coz I think it needs some restructuring - will be good to move this section into a new page, coz the current one is quite long already and has too many screenshots.

File: content/docs/studio/view-settings.md
Comment (tapadipti) [inline content/docs/studio/view-settings.md:1]: @shcheklein This is the page where I think it will be good to lead the users in case of non-dvc-repo errors. This page describes the different ways they can prepare their repo for viewing in Studio, and has a link to the DVC doc. Let me know what you think.

File: content/docs/studio/view-settings.md
Comment (tapadipti) [inline content/docs/studio/view-settings.md:79]: @shcheklein Let me know if you think we need to add more details regarding data remotes setup.

File: content/docs/studio/create-view.md
Comment (casperdcl) [inline content/docs/studio/create-view.md:21]: I don't think we need any of these `_v2` things

- we'd initially used `static.iterative.ai` because the images were WIP and we knew we'd change them
- now that they're finalised (I assume? #2469) we can put them in this repo as usual

CC @shcheklein related #93 <- #1115

File: content/docs/studio/create-view.md
Comment (casperdcl) [inline content/docs/studio/create-view.md:23]: other steps used imperative style so for consistency:

```suggestion
4. Specify additional connection settings if required.
```

File: content/docs/studio/create-view.md
Comment (casperdcl) [inline content/docs/studio/create-view.md:30]:
```suggestion
You should now see that a View has been added in your dashboard.
```

File: content/docs/studio/create-view.md
Comment (casperdcl) [inline content/docs/studio/create-view.md:49]:
```suggestion
`Share to Web`.
```

Comment (casperdcl) [reply content/docs/studio/create-view.md:49]: btw is "share to web" a better label than simply "share?"

"to web" implies there are other options ("by email?")

File: content/docs/studio/create-view.md
Comment (casperdcl) [inline content/docs/studio/create-view.md:54]:
```suggestion
GitHub app on the GitHub repositories/organizations that you want to use with
```

File: content/docs/studio/create-view.md
Comment (casperdcl) [inline content/docs/studio/create-view.md:71]:
```suggestion
> your GitHub organization. You can limit access to only repositories that
> you want to use with DVC Studio.
```

File: content/docs/studio/get-started.md
Comment (casperdcl) [inline content/docs/studio/get-started.md:16]: not sure about capitalisation consistency (summary view/summary View/Summary View)

File: content/docs/studio/index.md
Comment (casperdcl) [inline content/docs/studio/index.md:25]: is this related to CML or is it a Studio feature?

File: content/docs/studio/run-experiments.md
Comment (casperdcl) [inline content/docs/studio/run-experiments.md:11]:
```suggestion
> given below to run experiments directly from DVC Studio.
```

File: content/docs/studio/run-experiments.md
Comment (casperdcl) [inline content/docs/studio/run-experiments.md:48]:
```suggestion
At this point, the new experiment appears in the view table. If you just
committed to a new branch, then a new pull request will also have been created
from the new branch to the base branch.
```

File: content/docs/studio/run-experiments.md
Comment (casperdcl) [inline content/docs/studio/run-experiments.md:50]:
```suggestion
If your project is integrated with a CI/CD setup (e.g. GitHub Actions), the CI/CD
```

File: content/docs/studio/run-experiments.md
Comment (casperdcl) [inline content/docs/studio/run-experiments.md:55]:
```suggestion
set up CI/CD pipelines for your ML project, refer to [CML](https://cml.dev).
```

File: content/docs/studio/teams.md
Comment (casperdcl) [inline content/docs/studio/teams.md:42]: maybe best to tabulate this?

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:12]:
```suggestion
Alternatively, you could create views from:

- Non-DVC repositories
- Project sub-directories in a monorepo
- Custom files in your repository or remote/cloud storage

In each of these scenarios, you will need to specify advanced settings for DVC
Studio to be able to access the data required for visualization. Details are
given below.
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:16]:
```suggestion
DVC Studio creates Views by identifying datasets, metrics and hyperparameters
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:24]:
```suggestion
   data or huge models. These large assets reside in the cloud or other remote
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:25]: do we need to explain what DVC is over here? In which case there's probably already some other paragraph from elsewhere in the docs which we can copy-paste or link to.

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:32]:
```suggestion
   `dvc.yaml`. Each time you push a commit to this DVC repository, your view
   will reflect the new changes.
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:44]:
```suggestion
   with a non-DVC repository, you can still create views for it provided that
   metrics and hyperparameters are stored in CSV, JSON or YAML files. To
   visualize such custom data, simply
   [specify the custom files](#specifying-view-settings) to use, and DVC Studio
   will efficiently generate tables and plots for your custom input. For instance,
   if you have an ML project for which you generate and save metrics either manually
   or using some ML tracking tools, then you can create a view for this project by
   specifying the file (within your Git repo) which contains your saved metrics.
```

Comment (casperdcl) [reply content/docs/studio/view-settings.md:44]: or even briefer:

```suggestion
   with a non-DVC repository, you can still create views for it provided that
   metrics and hyperparameters are stored in CSV, JSON or YAML files. To
   visualize such custom data, simply
   [specify the custom files](#specifying-view-settings) to use, and DVC Studio
   will efficiently generate tables and plots for your custom input.
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:55]:
```suggestion
## Data remotes (cloud/remote storage)
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:58]:
```suggestion
present in a data remote (cloud storage or another location outside the Git
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:80]:
```suggestion
  will need to do it after your view has been created. First, create your
```

File: content/docs/studio/view-settings.md
Comment (casperdcl) [inline content/docs/studio/view-settings.md:85]:
```suggestion
  S3, GCP, etc.). Depending on the provider, you will be asked for more details
```

File: src/components/LayoutHeader/alert.tsx
Comment (casperdcl) [inline src/components/LayoutHeader/alert.tsx:11]: not sure if this needs to be `className` from `styled.module.css` or even an SVG `ReactComponent` CC @julieg18 @rogermparent

File: content/docs/start/index.md
Comment (tapadipti) [reply content/docs/start/index.md:82]: This will remain accessible.
@shcheklein can u pls confirm in case I am wrong.

File: content/docs/studio/create-view.md
Comment (tapadipti) [reply content/docs/studio/create-view.md:21]: The reason I used _v2 is because I needed the images uploaded for testing locally. But didn't want to mess up the prod versions until the PR was reviewed and ready.

File: content/docs/studio/create-view.md
Comment (tapadipti) [reply content/docs/studio/create-view.md:49]: `Share to web` is how the link appears on Studio. So, not changing it here. I think `Publish to web` may be a better option. But it needs to be changed in Studio first, before changing in the docs.

File: content/docs/studio/get-started.md
Comment (tapadipti) [reply content/docs/studio/get-started.md:16]: Since this is not a keyword, I guess we can leave it at 'summary view'? What do you think?

File: content/docs/studio/index.md
Comment (tapadipti) [reply content/docs/studio/index.md:25]: Studio, by virtue of CML. That is why the title for this section is "Use DVC, CML and Studio to:"

File: content/docs/studio/teams.md
Comment (tapadipti) [reply content/docs/studio/teams.md:42]: Ok, moving this to a follow-up ticket.

File: content/docs/studio/view-settings.md
Comment (tapadipti) [reply content/docs/studio/view-settings.md:16]: Not sure if the plural 'Views' is also a keyword for us. I think non-cap views would be ok. What do you think?

File: content/docs/studio/view-settings.md
Comment (tapadipti) [reply content/docs/studio/view-settings.md:25]: Yes. Will add.

File: content/docs/studio/view-settings.md
Comment (tapadipti) [reply content/docs/studio/view-settings.md:44]: I think the extra explanation may help some people. So, accepting your earlier suggestion.

File: content/docs/studio/create-view.md
Comment (tapadipti) [reply content/docs/studio/create-view.md:20]: Added to https://github.com/iterative/dvc.org/issues/2515 for removing _v2 from the image filenames (after launch).

File: src/components/LayoutHeader/alert.tsx
Comment (tapadipti) [reply src/components/LayoutHeader/alert.tsx:11]: Added to #2515 for fixing if needed.

File: content/docs/studio/teams.md
Comment (tapadipti) [reply content/docs/studio/teams.md:42]: Added to #2515

File: content/docs/studio/view-settings.md
Comment (tapadipti) [reply content/docs/studio/view-settings.md:28]: Added to #2515

**Comment by tapadipti** [comment]
@casperdcl thanks for your review. I've addressed your comments, and created a new ticket with pending issues. https://github.com/iterative/dvc.org/issues/2515

File: content/docs/studio/get-started.md
Comment (casperdcl) [reply content/docs/studio/get-started.md:16]: idk, I just noticed you'd been using capital "View" earlier so I assume it should be the same here.

**Review by casperdcl — APPROVED** [review]
approving to unblock merge (follow-up #2515)

File: src/components/LayoutHeader/alert.tsx
Comment (rogermparent) [reply src/components/LayoutHeader/alert.tsx:11]: While it looks like this works totally fine, the first iteration of this alert bar [did use a ReactComponent import](https://github.com/iterative/dvc.org/blob/809ac154ef02a8d9847432ae8ffd5bca666828f0/src/components/LayoutHeader/alert.tsx) to display a GitHub icon svg- everything else has been emojis/unicode IIRC.

An SVGR `ReactComponent` import seems like it would be more optimal, the difference is probably negligible to the point where is `ReactComponent` causes any problems (like with colors, for example, which didn't come into play on the GH svg), it's fine to stick with `img`.

File: content/docs/studio/index.md
Comment (shcheklein) [inline content/docs/studio/index.md:4]: why don't use the same simple language as we do in blog here:

```
DVC Studio is a web application that you can
[access online](https://studio.iterative.ai/) or even host on-prem. It works
with the data, metrics and hyperparameters that you add to your ML project
repositories.
```

Comment (tapadipti) [reply content/docs/studio/index.md]: Changed.

File: content/docs/start/index.md
Comment (shcheklein) [reply content/docs/start/index.md:82]: it's not a commit hash - it's a unique `view id`. It will stay stable until we remove that view.

File: content/docs/studio/create-view.md
Comment (shcheklein) [reply content/docs/studio/create-view.md:48]: yep, create a ticket on in the `iterative/viewer` please. Good cath @casperdcl .

File: content/docs/studio/create-view.md
Comment (shcheklein) [reply content/docs/studio/create-view.md:66]: now it's a comment here https://github.com/iterative/dvc.org/issues/2515

File: content/docs/studio/teams.md
Comment (shcheklein) [reply content/docs/studio/teams.md:42]: Agreed - it should be a matrix

File: content/docs/studio/view-settings.md
Comment (shcheklein) [reply content/docs/studio/view-settings.md:1]: I think it's complicated still, but it's fine to start here. Later we could use a page similar to this https://dvc-org-studio-docs-v2-kfehbw2.herokuapp.com/doc/user-guide/troubleshooting 

And a mechanism with redirects - https://error.dvc.org/lock-issue

File: content/docs/studio/view-settings.md
Comment (shcheklein) [reply content/docs/studio/view-settings.md:87]: it would be great to mention:

- we dont't read/access data
- we use it only to read plots/metrics files if they are not saved into Git. And people should use it only if they have this situation. They will see a specific message in their View.

File: content/docs/studio/create-view.md
Comment (tapadipti) [reply content/docs/studio/create-view.md:48]: Created https://github.com/iterative/viewer/issues/2098

File: content/docs/studio/get-started.md
Comment (shcheklein) [inline content/docs/studio/get-started.md:15]: I think `automatically` is not the best term here. It doesn't do anything automatically. It parses Git history and specific metafiles (or files you show to it). Automatically sounds like some magic is happening.

File: content/docs/studio/get-started.md
Comment (shcheklein) [inline content/docs/studio/get-started.md:4]: in the "DVC Studio creates views" view is not defined yet. Better to use some human readable terms - e.g. table with experiments? etc

File: content/docs/studio/get-started.md
Comment (shcheklein) [inline content/docs/studio/get-started.md:16]: data has a very specific meaning in ML, here the use of `data` is misleading

File: content/docs/studio/get-started.md
Comment (shcheklein) [reply content/docs/studio/get-started.md:20]: ` This includes tables` - why tables (plural?). To my mind view == table (with some filters applied).

File: content/docs/studio/view-settings.md
Comment (tapadipti) [reply content/docs/studio/view-settings.md:87]: Adding this: "Note that DVC Studio uses the credentials only to read plots/metrics files if they are not saved into Git. It does not access any other data in your remote storage. And you do not need to provide the credentials if any DVC data remote in not used in your Git repository."

File: content/docs/studio/view-settings.md
Comment (shcheklein) [inline content/docs/studio/view-settings.md:24]: DVC is not only about not bloating Git repo. It's about pipelines, connecting then with CML, etc. We should be explaining these benefits better I think

File: content/docs/studio/view-settings.md
Comment (tapadipti) [reply content/docs/studio/view-settings.md:1]: Added to #2515

File: content/docs/studio/get-started.md
Comment (tapadipti) [reply content/docs/studio/get-started.md:4]: Will add a link from 'views' to the 'What is a view?' section on that page. Works?

File: content/docs/studio/get-started.md
Comment (tapadipti) [reply content/docs/studio/get-started.md:16]: changing 'The data' to 'These details'

File: content/docs/studio/view-settings.md
Comment (tapadipti) [reply content/docs/studio/view-settings.md:24]: We should have a separate section/paragraph on DVC and its benefits, and link it to the DVC doc. For now, I'm adding this to the current paragraph: "DVC also
   enables you to
   [share your data and model files](/doc/use-cases/sharing-data-and-model-files),
   [create data registries](/doc/use-cases/data-registries),
   [create data pipelines](/doc/start/data-pipelines), connect them with
   [CML](/doc/cml) for CI/CD in machine learning, and so on. Find more about the
   features and benefits of DVC [here](/doc/start)."

Comment (shcheklein) [reply content/docs/studio/view-settings.md:24]: ideally we should zoom out and summarize high level benefits here (e.g. pipelines == reproducibility).

One way can be - DVC goes beyond that. It lets you organize your project in a reproducible way, it gives you way to save your experiments holistically as Git commits - data, models, metrics, params ... You could use it to organize manage your data using Git flow (data registries) ... 

something like this ...

File: content/docs/studio/get-started.md
Comment (shcheklein) [reply content/docs/studio/get-started.md]: pls check other places

how about Values ...?

File: content/docs/studio/get-started.md
Comment (tapadipti) [reply content/docs/studio/get-started.md]: removed

**Review by shcheklein — APPROVED** [review]
Approving this, so that it can be merge before the release

File: content/docs/studio/get-started.md
Comment (casperdcl) [reply content/docs/studio/get-started.md]: "intelligently?"

File: content/docs/studio/get-started.md
Comment (casperdcl) [reply content/docs/studio/get-started.md]: "This info?"

File: src/components/LayoutHeader/alert.tsx
Comment (tapadipti) [reply src/components/LayoutHeader/alert.tsx:11]: @rogermparent There was some issue when we used `img`. We were getting Cloudflare alert that the page was offline. It was not breaking all the time, but Ivan noticed it when he was trying to load the page. We replaced the svg img with 🚀 in this [PR](https://github.com/iterative/dvc.org/pull/2532/files) and the alerts went away (page stopped breaking). Any idea why that may have happened and how we can use images if needed?

Comment (shcheklein) [reply src/components/LayoutHeader/alert.tsx:11]: @tapadipti let's please create a proper issue/bug with the screenshot, what we have done to fix, etc

Comment (jorgeorpinel) [reply src/components/LayoutHeader/alert.tsx:11]: https://github.com/iterative/dvc.org/issues/2540

**Pull request merged**
